source/common/CMakeLists.txt

```diff
@@ -19,9 +17,31 @@ target_include_directories(ur_common PUBLIC
     ${CMAKE_SOURCE_DIR}/include
 )
 
+message(STATUS "Download Unified Memory Framework from github.com")
```

Comment:
in oneapi we have a switch to enable building against a local checkout of unified runtime. Should we have the same thing here?

c.f [intel/llvm/139b13172](https://github.com/intel/llvm/commit/139b1317278fc777aa36f71096d6639fe6fd3ff0)

Reply:
I don't think there will be many changes required in the UR when bumping UMF version in the future. 

However, we've had some discussion about potentially moving some of the memory provider logic for L0 and CUDA to UMF - if we decide to do that, then adding such a switch might a good idea.